app/views/checkouts/_checkout.html.erb

```diff
@@ -27,7 +27,9 @@
   <div class="collapse w-100" id="collapseExample-<%= checkout.key.parameterize %>">
     <dl class="row justify-content-center">
       <dt class="col-5">Borrowed on:</dt>
```

Comment:
Stupid nitpick for @jvine; if we're showing the time, should this be `Borrowed at` or some such?

Reply:
Alternative would be to remove the word "on" globally. So it's just "Borrowed: July 15, 2019" or "Borrowed:  Today at 4:29pm"  

(The "on" has been bugging me anyway.)

Reply:
👍 ; @jkeck, would you mind doing that as part of this PR?